From 542587bae6b894bdd1a5cfd38223f8054d166fc3 Mon Sep 17 00:00:00 2001
From: Adrian Perez de Castro <aperez@igalia.com>
Date: Sat, 10 Apr 2021 18:28:43 +0300
Subject: [PATCH 09/13] Revert "ws: Call wl_client_destroy for the created
 wl_client in Instance::unregisterViewBackend()"

This reverts commit 9a0e6cb62bfae44ef3496d950899ecb34ddd24a5.
---
 src/ws.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/ws.cpp b/src/ws.cpp
index 80cb39b..87faaf3 100644
--- a/src/ws.cpp
+++ b/src/ws.cpp
@@ -534,7 +534,6 @@ void Instance::unregisterViewBackend(uint32_t bridgeId)
     auto it = m_viewBackendMap.find(bridgeId);
     if (it != m_viewBackendMap.end()) {
         it->second->apiClient = nullptr;
-        wl_client_destroy(wl_resource_get_client(it->second->resource));
         m_viewBackendMap.erase(it);
     }
 }
-- 
2.31.1

